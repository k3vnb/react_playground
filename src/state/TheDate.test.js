import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import TheDate from './TheDate';

describe('theDate component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<TheDate />, div);
        ReactDOM.unmountComponentAtNode(div);
    })

    it.skip('renders in the UI as expected', () => {
        const tree = renderer
            .create(<TheDate />, )
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})


