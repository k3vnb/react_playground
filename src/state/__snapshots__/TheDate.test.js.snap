// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theDate component renders in the UI as expected 1`] = `
<div>
  5/30/2019, 6:37:36 PM
</div>
`;
